src/Sylius/Bundle/CoreBundle/Test/Factory/TestUserFactoryInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Test\Factory;
+
+use Sylius\Component\Core\Model\UserInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface TestUserFactoryInterface
+{
+    const DEFAULT_USER_FIRST_NAME = 'John';
```

Comment:
Why do you define some variables in interface and other in a class implementation?